scripts/just.config.js

```diff
@@ -62,10 +64,12 @@ module.exports = function preset() {
   task('check-for-modified-files', checkForModifiedFiles);
   task('generate-version-files', generateVersionFiles);
   task('generate-package-manifest', generatePackageManifestTask);
-  task('ts', () => {
+  task('ts:compile', () => {
```

Comment:
You should check whether there are any projects which redefine `ts` and will also need to be updated (I think that's more of an issue with `build` but worth checking)

Reply:
I did check. The only other use is in [monaco editor](https://github.com/OfficeDev/office-ui-fabric-react/blob/f5dd56da0135565d81ab8396ba553210b1e0f810/packages/monaco-editor/just.config.js#L26). However, it seems to do its own thing in general. Can [transformDtsTask](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/monaco-editor/tasks/transformDtsTask.js) be removed with this change?

Reply:
Yes that package is...interesting (says its author 😆). transformDtsTask can be removed if you update your postprocess task to take configurable paths, since not everything in monaco-editor is under lib.

Reply:
I think I'll leave that to another PR as an improvement. Since monaco-editor is taking over the whole `ts` task (including dts postprocessing), I think it's ok to leave this PR as-is.